src/System.Management.Automation/singleshell/config/MshSnapinInfo.cs

```diff
@@ -1010,13 +1014,20 @@ internal static Collection<PSSnapInInfo> ReadEnginePSSnapIns()
 
             // create default mshsnapininfo objects..
             Collection<PSSnapInInfo> engineMshSnapins = new Collection<PSSnapInInfo>();
+            string assemblyVersionString = assemblyVersion.ToString();
```

Comment:
This variable is used only once. Maybe remove this variable and pass in `assemblyVersion.ToString()` as before?

Reply:
I moved it from `for` cycle.